FirebaseAuth/Tests/Sample/Sample/ApplicationDelegate.m

```diff
@@ -45,6 +47,11 @@ - (BOOL)application:(UIApplication *)application
   [[FIRConfiguration sharedInstance] setLoggerLevel:FIRLoggerLevelInfo];
 
   // Configure the default Firebase application:
+  FIRAppCheckDebugProviderFactory *providerFactory =
```

Comment:
I would remove Sample app changes for now (and just leave it in your local workspace for testing).

Reply:
removed. thx!